transit_odp/publish/views/timetable/list.py

```diff
@@ -88,3 +88,26 @@ def render_to_response(self):
         response = HttpResponse(file_, content_type="text/csv")
         response["Content-Disposition"] = f"attachment; filename={csv_filename}"
         return response
+
+
+class SeasonalServiceView(OrgUserViewMixin, SingleTableView):
+    template_name = "publish/seasonal_services/index.html"
+    model = SeasonalService
+    table_class = SeasonalServiceTable
+    paginate_by = 10
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        org_id = self.kwargs["pk1"]
+        context["pk1"] = org_id
+        context[
+            "seasonal_services_counter"
+        ] = SeasonalService.objects.get_count_in_organisation(org_id)
+        return context
+
+    def get_table_data(self, **kwargs):
+        # org_id = self.request.GET.get("pk1")
```

Comment:
Please delete this commented-out code

Reply:
deleted in refactor commit. Probably should have put that in the view commit!